src/Tag/Tag.tsx

```diff
@@ -40,6 +40,7 @@ const Wrapper = styled.div<IWrapper>`
 `
 
 const TagText = styled(Text)`
-  font-family: 'Gordita', sans-serif;
+  font-family: 'Circular', sans-serif;
```

Comment:
I think you can just get rid of this since `Circular` should be the default font family anyway